testsupport/tiers/checks.go

```diff
@@ -25,13 +25,14 @@ import (
 
 const (
 	// tier names
-	appstudio       = "appstudio"
-	appstudiolarge  = "appstudiolarge"
-	appstudioEnv    = "appstudio-env"
-	base            = "base"
-	base1ns         = "base1ns"
-	base1ns6didler  = "base1ns6didler"
-	base1nsnoidling = "base1nsnoidling"
+	appstudio         = "appstudio"
+	appstudiolarge    = "appstudiolarge"
+	appstudioEnv      = "appstudio-env"
+	base              = "base"
+	base1ns           = "base1ns"
+	base1nsGoTemplate = "base1ns-gotemplate"
```

Comment:
can you clean it up?

Reply:
The whole purpose of the `CustomNSTemplateTier` is to reuse the already existing tiers and checks so we don't have to modify the `checks.go` file for every single custom tier that we define/use in any of the tests.
Apart from that, if what you have said was true, then the existing tests for other custom tier would fail, wouldn't they?

Reply:
this was the reason the other custom tiers test was passing,